pkg/log/logger.go

```diff
@@ -116,14 +126,17 @@ func createConfig() *zap.Config {
 }
 
 func SetLogLevel(newLevel string) error {
+	logger := Logger()
+	config := GetConfig()
+
 	oldLevel := config.Level.String()
 
 	// noop if the input is the same as what is set
 	if newLevel == oldLevel {
 		return nil
 	}
 
-	logger.Info("Updating log level",
+	Logger().Info("Updating log level",
```

Comment:
consistency: either use  `Logger()` or use the local var logger throughout the function:  line 129, 139 & 147

Reply:
good catch, this should be logger from Line 129.